notebook.ipynb

```diff
@@ -54,13 +54,13 @@
    "execution_count": null,
```

Comment:
<p>Should be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-call-center/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
yes I forgot to delete those lines